packages/panzer/adapters-stk/example/MixedCurlLaplacianExample/Debugging.nb

```diff
@@ -0,0 +1,1741 @@
+(* Content-type: application/vnd.wolfram.mathematica *)
```

Comment:
did you mean to commit this file?

Reply:
I can remove this file.

Reply:
Ok, I removed it. This needs to be re-approved.